build.gradle

```diff
@@ -111,6 +111,7 @@ dependencies {
     compile 'ch.qos.logback:logback-classic:1.2.3'       // Logger
     compile 'com.google.apis:google-api-services-cloudbilling:v1-rev20190805-1.30.1'
     compile 'com.google.apis:google-api-services-cloudresourcemanager:v1-rev549-1.25.0'
+    compile 'com.google.apis:google-api-services-iam:v1-rev247-1.23.0'
```

Comment:
I think we don't actually need this dependency since we ended up using the cloud resource manager api for the access.